build/templates/backend/Makefile

```diff
@@ -2,6 +2,16 @@
 ARCHIVE := $(shell basename {{.Backend.BinaryURL}})
 
 all:
+	mkdir backend
+{{- if ne .Backend.DockerImage "" }}
+	docker container inspect extract > /dev/null 2>&1 && docker rm extract || true
+	docker create --name extract {{.Backend.DockerImage}}
```

Comment:
create docker extraction image to allow copying of pre-built contents directly from the official docker image